addon-test-support/should-force-audit.ts

```diff
@@ -1,17 +1,41 @@
-import { ENABLE_A11Y_AUDIT } from './cli-options';
+import { ENABLE_A11Y_AUDIT as ENABLE_A11Y_AUDIT_ENV } from './cli-options';
```

Comment:
Not a huge fan of doing this, as it makes it more confusing to read when the name changes to a local alias in certain files. Can we figure out better names so we don't have to alias?

Reply:
Yeah, no problem.